msg/RadarPoint.msg

```diff
@@ -0,0 +1,10 @@
+# All variables below are relative to the radar's frame of reference.
+# This message is not meant to be used alone but as part of a stamped or array message.
+
+float32 range                            # Distance (m) from the sensor to the detected point.
+float32 angle                            # Angle (in radians) in the azimuth plane between the sensor and the detected point.
+                                         #   Negative angles are anticlockwise from the sensor.
+float32 elevation                        # Angle (in radians) in the elevation plane between the sensor and the detected point.
```

Comment:
This name is not clear that it's angular you have to read the comments. Elevation could easily be euclidian. A quick search shows that the angle is sometimes called elevation and sometimes altitude, but both refer to the angle above the horizon. To that end I think the `elevation_angle` would be best to disambiguate. And if this ends up as `elevation_angle` the azimuth angle should probably be fully written out.

Reply:
I think in the context with changing the other to `azimuth`, `elevation` is entirely clear for anyone with a geospatial/radar background. They're peas in a pod terms that always appear together with a specific meaning.

Reply:
I vote to go with azimuth and elevation